ladybug/src/main/java/nl/nn/ibistesttool/PipeLineSessionDebugger.java

```diff
@@ -55,9 +55,18 @@ public Object invoke(Object self, Method method, Method proceed, Object[] args)
 			putAll((Map<String,Object>)args[0]);
 			return null;
 		}
+		if(method.getName().equals("getMessage")) {
+			return getMessage((String)args[0]);
+		}
 		return method.invoke(pipeLineSession, args);
 	}
 
+	private Object getMessage(String name) {
+		Object value = pipeLineSession.getMessage(name);
+		ibisDebugger.showValue(pipeLineSession.getMessageId(), "SessionKey "+name, value);
+		return value;
+	}
```

Comment:
it might be better to use the `get` method instead (getMessage calls that one directly) so that we you get every occurrence.

Reply:
Using get results in excessive/double checkpoints in ladybug. See 
[Pipeline CallApiListener (1).zip](https://github.com/ibissource/iaf/files/8831644/Pipeline.CallApiListener.1.zip)

On the other hand, getMessage is the commonly used method to resolve a sessionKey in pipes